admin/aioseop_module_class.php

```diff
@@ -1770,6 +1767,7 @@ function admin_enqueue_styles( $hook_suffix ) {
 		public function admin_enqueue_scripts( $hook_suffix ) {
 			wp_enqueue_script( 'sack' );
 			wp_enqueue_script( 'jquery' );
+			wp_enqueue_script( 'jquery-ui-tabs' );
```

Comment:
Is `jquery-ui` no longer required?

Reply:
according to https://developer.wordpress.org/reference/functions/wp_enqueue_script/ it does not seem to be a needed dependency